tools/detect_targets.py

```diff
@@ -92,6 +99,23 @@ def main():
         traceback.print_exc(file=sys.stdout)
         print "[ERROR] %s" % str(exc)
         sys.exit(1)
+        
+def get_interface_version(mount_point):
+    """ Function returns interface version from the target mounted on the specified mount point
+            mount_point = mut['port']
```

Comment:
What is this line of the function description supposed to tell me?

Reply:
Example usage. Though I just found one mistake- let me push an update there. An accurate usage should be mount_point = mut['disk']